cosmos/cosmoshub-validator-delegations/package.json

```diff
@@ -4,24 +4,22 @@
   "repository": "https://github.com/graphprotocol/example-subgraph",
   "license": "MIT",
   "scripts": {
-    "prepare:cosmoshub": "mustache config/cosmoshub.json subgraph.template.yaml > subgraph.yaml",
-    "prepare:osmosis": "mustache config/osmosis.json subgraph.template.yaml > subgraph.yaml",
     "codegen": "graph codegen",
     "build": "graph build",
-    "create-local": "graph create cosmos-validator-delegations --node http://localhost:8020",
-    "deploy-local": "graph deploy cosmos-validator-delegations -l v0.1.0 --ipfs http://localhost:5001 --node http://localhost:8020",
-    "remove-local": "graph remove cosmos-validator-delegations --node http://localhost:8020",
+    "create-local": "graph create cosmos-validator-delegations --node http://127.0.0.1:8020",
+    "deploy-local": "graph deploy cosmos-validator-delegations -l v0.1.0 --ipfs http://127.0.0.1:5001 --node http://127.0.0.1:8020",
+    "remove-local": "graph remove cosmos-validator-delegations --node http://127.0.0.1:8020",
     "create": "graph create cosmos-validator-delegations --node https://api.thegraph.com/deploy/",
     "deploy": "graph deploy cosmos-validator-delegations --ipfs https://api.thegraph.com/ipfs/ --node https://api.thegraph.com/deploy/"
```

Comment:
Let's use `cosmoshub-validator-delegations` as the name everywhere (including the name of the NPM package).